README.md

```diff
@@ -1,24 +1,73 @@
-The website for Urbit.
+The website for Urbit is maintained by [Tlon](https://tlon.io) and the Urbit community. Issues and contributions are welcome.
```

Comment:
It’s maintained by the Urbit Foundation actually.

Reply:
should Tlon be removed from the second line of the docs [index](https://urbit.org/docs) as well?